stack.yaml

```diff
@@ -8,7 +8,7 @@ packages:
 extra-deps:
 - Rasterific-0.6.1.1
 - FontyFruity-0.5.3.2
-- sdl2-2.1.3
+- sdl2-2.1.3.1
```

Comment:
I had to slightly bump this in order to match the corresponding bump to vector-0.12 caused by the above resolver bump.